### PR TITLE
21 display aqi information from api

### LINKED ISSUE #21: Display AQI information from API
**Is your feature request related to a problem? Please describe.**
As a user, I should be able to see correct air quality information according to the city I searched for.

**Describe the solution you'd like**
Name of the city and the AQI number should be dynamically displayed on the page.

**Additional context**
Data is being pulled from the aqicn.org.

### PR DESCRIPTION
**Pull request by lindakw**
Made a few small changes to dynamically add the city name and also the AQI number in the main circle display.

### CODE REVIEW

File: screens/aqi_screen/aqiPage.tsx [hunk 1]
```diff
@@ -46,13 +49,13 @@ const Aqi: FC<AQIScreenProps> = (props) => {
...
         <Text style={styles.title}>AirOMeter</Text>
-        <Text style={styles.cityName}>Fresno</Text>
+        <Text style={styles.cityName}>{cityName}</Text>
 
 
         <View style={styles.aqiCircle}>
           <View style={styles.aqiTextContainer}>
             <Text style={styles.aqiTextTitle}>AQI</Text>
-            <Text style={styles.aqiTextNum}>35</Text>
+            <Text style={styles.aqiTextNum}>{aqiInfo.aqi}</Text>
```
Comment (kmausolf) [inline screens/aqi_screen/aqiPage.tsx:58]: You may want to check if `aqiInfo` is null before trying to index into it.  You could prevent this by doing something like:

```
aqiInfo ? aqiInfo.aqi : "N/A"
```

**Review by kmausolf — COMMENTED** [review]
Nice work on getting the redux data in here!  I have one suggestion that could help prevent a crash.

**Review by Kacique — APPROVED** [review]
Good Work

File: screens/aqi_screen/aqiPage.tsx (hunk 1 above)
Comment (lindakw) [reply screens/aqi_screen/aqiPage.tsx:58]: awesome! I'll add that in! thanks!

**Pull request merged**